docs/modules/ROOT/pages/discovery.adoc

```diff
@@ -0,0 +1,50 @@
+:clusterName: simple-hive-derby
+:namespace: stackable
+:metastorePort: 9083
```

Comment:
Can we use some more obvious placeholder names?

Reply:
like prefixing with `hive` or what do you suggest?

Reply:
Using something like `$CLUSTERNAME` as the cluster name.

Reply:
Looks like it works without generating extra warnings if I declare `:clusterName: \{clusterName\}` (and so on).

Reply:
I'm not sure if I'm following the discussion correctly here, is it about if the exapmle should be this:
```
apiVersion: hive.stackable.tech/v1alpha1
kind: HiveCluster
metadata:
  name: {clusterName} # <1>
  namespace: {namespace} # <2>
spec:
...
```

vs this:

```
apiVersion: hive.stackable.tech/v1alpha1
kind: HiveCluster
metadata:
  name: simple-hive # <1>
  namespace: default # <2>
spec:
  ...
```

Reply:
Yeah, but mostly in the context of the discovery CMs, not the cluster definition itself.

Reply:
So [here](https://github.com/stackabletech/hive-operator/pull/145/files#diff-b83490b212c5dfb5449c7b6df704ebe41c33ed4fe36223a1df2ad7399b968bb1R37) ?

Reply:
Yep, and the URLs listed below that.

Reply:
Yeah if we use the abstract names we should adapt the examples of course as well. I like it explicit (and thats how it is in opa and kafka as well).

Reply:
Err, the abstract version is explicit about where the values are coming from. The _concrete_ version only gives a weak implication about the relationship.

Reply:
Can we do something like this as a compromise: 

> The resulting discovery ConfigMap follows the naming convention  `{namespace}/{clusterName}`, which for the example shown above would resolve to 'stackable/simple-hive-derby' ...

Reply:
I think Teo refers to the explicit names in the connection strings:
```
`HIVE`::
====
Contains the thrift protocol connection string for the Hive metastore:
[subs="attributes"]
  thrift://{clusterName}-metastore-{roleGroup}-0.{clusterName}-metastore-{roleGroup}.{namespace}.svc.cluster.local:{metastorePort}
```
should stay as is (without the replacements)

Reply:
Ah! Sorry for being daft today (and others days as well I guess).

I think it is better with the abstract placeholders in there, yes. Same idea if we wanted to make it explicit: generate a concrete example on top of the abstract version, but if we have to pick one of the two I think it should be the abstract one.